src/goto-instrument/code_contracts.cpp

```diff
@@ -168,23 +169,56 @@ void code_contractst::add_quantified_variable(
   replace_symbolt &replace,
```

Comment:
Could we use `const exprt& expression`?

Reply:
Currently, I do not think that it is possible. But I the medium/long term, we should get rid of this parameter entirely.